doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1307 @@
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This release introduces significant, potentially breaking, changes to the SDK:
+
+* The `previous method to define a board (hardware model)`_ is deprecated and being replaced by :ref:`a new method (hardware model v2) <zephyr:board_porting_guide>`.
+* The previous method to define :ref:`multi-image builds (parent-child images) <ug_multi_image>` is deprecated and being replaced by :ref:`zephyr:sysbuild`.
+
+All samples and applications in the SDK have been migrated.
+Applications that use custom defined boards should not be affected and can use the deprecated methods in this and the next release.
+nRF Connect for VS Code users migrating to the latest version of the SDK might be affected.
+
+The deprecated methods are scheduled for removal after the next release.
+We recommend transitioning to the alternatives as soon as possible.
+Consult migration guides for <hardware model v2> and <sysbuild> TBD.
+Exercise caution when migrating production environments to the latest SDK.
+
+Added the following features as supported:
+
+* Bluetooth® LE:
+
+  * Path loss monitoring in the SoftDevice controller, which is part of LE Power Control.
+    This feature does not yet have a host stack API, it requires access via HCI commands.
+
+* Matter:
+
+  * Matter 1.3, which enables energy reporting for Matter devices and support for water and energy management, electric vehicle charges, and new major appliances.
+    Learn more about Matter 1.3 in the `Matter 1.3 CSA blog post`_.
+
+* Wi-Fi®:
+
+  * Wi-Fi and Thread coexistence.
+  * Software-enabled Access Point (SoftAP) for provisioning that was introduced in |NCS| v2.6.0 as experimental is now supported.
+  * Raw Wi-Fi reception in both Monitor and Promiscuous modes that was introduced in |NCS| v2.6.0 as experimental is now supported.
+  * New samples: :ref:`softap_wifi_provision_sample`, :ref:`wifi_promiscuous_sample`, and :ref:`wifi_thread_coex_sample`.
+
+* Other:
+
+  * Google Find My Device that allows the creation of a locator device that is compatible with the Android `Find My Device app`_.
+    This feature is showcased in the :ref:`fast_pair_locator_tag` sample.
+  * Hardware model v2 (HWMv2), an improved extensible system for defining boards. This is the default boards definition system from this |NCS| release and onwards.
+    TBD add a link to the migration guide.
+  * :ref:`zephyr:sysbuild`, an improved and extensible system for multi-image build, replacing :ref:`ug_multi_image` (parent/child images).
+    Link to migration guide TBD.
+
```

Comment:
Can we add a statement about IPC Radio here? I think it's important for the customers to know that this is now the default secondary image for samples and application that use short-range radio.
"_Samples and application that use short-range and run on multi-core SoCs were migrated to use the :ref:`ipc_radio` as the default image for network/radio core_".